release/src/main/scripts/cut_release_branch.sh

```diff
@@ -127,10 +124,6 @@ echo "==================Current working branch======================="
 echo ${RELEASE_BRANCH}
 echo "==============================================================="
 
-sed -i -e "s/${DEV}/${RELEASE}/g" gradle.properties
-sed -i -e "s/${DEV}/${RELEASE}/g" sdks/python/apache_beam/version.py
-sed -i -e "s/${DEV}/${RELEASE}/g" sdks/go/pkg/beam/core/core.go
```

Comment:
To clarify, these are set when building the RC now ?

Reply:
Yes. These will actually be set when creating the RC tag.